pkg/client/e2e.go

```diff
@@ -44,9 +45,9 @@ func (*SonobuoyClient) GetTests(reader io.Reader, show string) ([]reporters.JUni
 			}
 			// TODO(chuckha) consider reusing this function for any generic e2e-esque plugin results.
 			// TODO(chuckha) consider using path.Join()
-			if path == e2eJunitPath {
+			if path == e2eJunitPath || path == legacye2eJunitPath {
```

Comment:
Since the old path is at a parent directory from the new one (and so a user can't actually write data to that file that could be misleading) it is fine to just search for either of the paths.